src/ArduinoIoTCloudLPWAN.cpp

```diff
@@ -0,0 +1,208 @@
+/*
+   This file is part of ArduinoIoTCloud.
+
+   Copyright 2019 ARDUINO SA (http://www.arduino.cc/)
+
+   This software is released under the GNU General Public License version 3,
+   which covers the main part of arduino-cli.
+   The terms of this license can be found at:
+   https://www.gnu.org/licenses/gpl-3.0.en.html
+
+   You can be released from the requirements of the above licenses by purchasing
+   a commercial license. Buying such a license is mandatory if you want to modify or
+   otherwise use the software for commercial activities involving the Arduino
+   software without disclosing the source code of your own applications. To purchase
+   a commercial license, send an email to license@arduino.cc.
+*/
+
+#if defined(ARDUINO_SAMD_MKRWAN1300) || defined(ARDUINO_SAMD_MKRWAN1310)
+
+#include<ArduinoIoTCloudLPWAN.h>
+
+#ifdef ARDUINO_ARCH_SAMD
+  #include <RTCZero.h>
+  RTCZero rtc;
+#endif
+
+ArduinoIoTCloudLPWAN::ArduinoIoTCloudLPWAN() :
+  _connection(NULL) {}
+
+ArduinoIoTCloudLPWAN::~ArduinoIoTCloudLPWAN() {
+}
+
+int ArduinoIoTCloudLPWAN::connect() {
+  _connection->connect();
+  int state = _connection->getStatus() == NetworkConnectionState::INIT ? 1 : 0;
```

Comment:
Minor improvement suggestion: Since state is written-once and read-only after you can make it `const`.

Reply:
fixed